pkg/oci/containerd.go

```diff
@@ -294,13 +294,11 @@ func (c *Containerd) GetBlob(ctx context.Context, dgst digest.Digest) (io.ReadCl
 	if err != nil {
 		return nil, err
 	}
-	return struct {
-		io.Reader
-		io.Closer
-	}{
-		Reader: content.NewReader(ra),
-		Closer: ra,
-	}, nil
+	rs, err := newHTTPReadSeeker(logr.FromContextOrDiscard(ctx), ra)
```

Comment:
The `bytes.Reader` implements `Seek`. We could use it instead which would remove the need for our own seeker implementation.

https://pkg.go.dev/bytes#Reader.Seek